exercise1.js

```diff
@@ -40,17 +71,41 @@
 // Problem 4:
 // Write a JavaScript function with a function that reverses your telephone number. Use the split() and join() methods from the previous problem as well as the toString() method to convert a number into a string and reverse() method to reverse an array in place.
 
-//your code...
+const reverseNumber = (number) => {
+    let stringNumber = number.toString();
+    let arrayNumber = stringNumber.split("");
+    arrayNumber = arrayNumber.reverse();
+    stringNumber = arrayNumber.join("");
+    console.log(stringNumber);
+    
+}
+reverseNumber(6173692334);
 
 
 
 
 
 /************************************************************* */
 // Problem 5:
-// Write a JavaScript function that creates a car object using information about your car. Include the make, model, year, and color. Write a function to get the year, color, make, and model in that order.
+// Write a JavaScript function that creates a car object using information about your car. Include the make, model, year, and color. 
+//Write a function to get the year, color, make, and model in that order.
+
+const car = class {
+    constructor(make,model,year,color){
+        this.make = make;
+        this.model = model;
+        this.year = year;
+        this.color = color;
+    }
+    carType(){
+        console.log(this.year+" "+this.color+" "+this.make+" "+this.model);
+    }
+    
+
+}
```

Comment:
this definitely works, but the instructions wasn't requiring a class, but a simple function.  

technically, a class is a syntactic sugar on top of constructor functions, but this problem did not even require the usage of constructor functions.  

You still solved it however.